ocs_ci/templates/ocs-deployment/multicluster/odf_multicluster_orchestrator.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: odf-orchestrator-operatorgroup
+  namespace: openshift-operators
+---
+apiVersion: operators.coreos.com/v1alpha1            
+kind: Subscription       
+metadata:
+  name: odf-orchestrator-sub
+  namespace: openshift-operators    
+spec:
+  channel: stable-4.9      
+  name: odf-multicluster-orchestrator
+  source:  ocs-catalogsource
+  sourceNamespace: openshift-marketplace
+  currentCSV: odf-multicluster-orchestrator.v4.9.0-120.ci
```

Comment:
Specific version mentioned here is old now. Needs to be updated.

Reply:
I can add code for version modification in the next PR.

Reply:
marking as TODO